### ISSUE TITLE
rspec failures

### ISSUE THREAD

**Issue by dwmarshall** [burtlo/ark#118]
Among others:

```
  1) ark install installs
     Failure/Error: expect(resource).to notify('execute[unpack /var/chef/cache/test_install-2.tar.gz]').to(:run)
       expected "directory[/usr/local/test_install-2]" to notify "execute[unpack /var/chef/cache/test_install-2.tar.gz]" with action :run, but did not.
       
       Other notifications were:
       
           "directory[/usr/local/test_install-2]" notifies "execute[unpack /var/folders/jb/3mjvb9r133v0hzqfyn20yz34000zj7/T/d20151028-53172-ul6izb/test_install-2.tar.gz]" to :run, :delayed
        
     # ./spec/resources/default_spec.rb:19:in `block (3 levels) in <top (required)>'
```

These lines in `spec/resources/default_spec.rb` are intended to correct that:

```
  before(:each) do
    Chef::Config[:file_cache_path] = "/var/chef/cache"
  end
```

but my Chef-fu is not strong enough to know how to fix that.

**Comment by dwmarshall** [comment]
@burtlo any ideas of what's going on? I'm using ChefDK 0.8.1

**Comment by samdunne** [comment]
In your spec_helper.rb you need to specify:
`config.file_cache_path = '/var/chef/cache'`

It doesn't seem to work otherwise.

**Comment by samdunne** [comment]
Fixed with #119

**Comment by dwmarshall** [comment]
Thanks @samdunne, your work on #119 is the basis for a PR I'm going to have finished tonight I think. So, I'm going to close this issue now.
